SplunkRumWorkspace/SplunkRum/SplunkRum/Session.swift

```diff
@@ -62,6 +68,15 @@ func getRumSessionId() -> String {
     for callback in callbacks {
         callback()
     }
-
+    if isSessionIdChanged {
+        createSessionIdChangeSpan()
+    }
     return rumSessionId
 }
+func createSessionIdChangeSpan() {
+    isSessionIdChanged = false
+    let tracer = buildTracer()
+    let span = tracer.spanBuilder(spanName: "sessionId.change").startSpan()
```

Comment:
i add start time and and time.

Reply:
`isSessionIdChanged` is not functional - local variable and it is not static as well, it is used by more than one function of the same swift file.

Reply:
> `isSessionIdChanged` is not function - local variable and it is not static as well because it is used by more than one function of the same swift file.

That variable does not need to be file-level.  Please scope it locally to `getRumSessionId` and think through the logic.   It does not need to be assigned in `createSessionIdChangeSpan`.

Reply:
make  `isSessionIdChanged` local variable and changed code  accordingly.